### ISSUE TITLE
Windows shortcuts which point to executables can't be executed

### ISSUE THREAD

**Issue by JonathanBerkeley** [nushell/nushell#13327]
### Describe the bug

Windows shortcuts (.lnk files) behave unusually in Windows Nushell. I have the folder which contains the shortcut "n.lnk" in my Windows Path. This shortcut points to notepad++.exe. In PowerShell I am able to write `start n` to start notepad++, and also use it as part of bigger expressions.

Firstly, with `exec`:
![image](https://github.com/nushell/nushell/assets/35268549/7ba2142b-8df5-4a18-8736-7be017c526ba)

```rs
Error: nu::shell::external_command
  × External command failed
   ╭─[entry #1:1:1]
 1 │ exec n.lnk
   · ──┬─
   ·   ╰── Error spawning external command
   ╰────
  help: %1 is not a valid Win32 application. (os error 193)
```

The shortcut file:
![image](https://github.com/nushell/nushell/assets/35268549/ef66a21b-8ad2-4926-ade6-25df32a5359a)

It works if I use the full path to the shortcut in an expression, but not if I use the shortcut abbreviated (but it's not a Path issue, because if it doesn't find the shortcut it's a different error, as in the case of x.lnk in below screenshot which doesn't exist):
![image](https://github.com/nushell/nushell/assets/35268549/41c907db-7fbb-46e6-a044-c79ff65a9b1a)

Comparatively, both of the following work in PowerShell:
![image](https://github.com/nushell/nushell/assets/35268549/746b0437-3b6b-48d2-b4f7-80c0ce56d0e1)
![image](https://github.com/nushell/nushell/assets/35268549/fb9efb92-c2f1-4945-aa8f-7750237bdb78)

Same for cmd and some other terminals.

### How to reproduce

1. Create a Windows shortcut that points to an executable, like notepad or similar
2. Add the folder where the shortcut is located to your Windows Path.
3. Start a new terminal such as PowerShell or cmd, you can now use `start <shortcut name>.lnk`
4. Start a new nushell instance, `exec <shortcut name>.lnk` will fail
5. Use the shortcut as part of an expression, E.G:
`glob "**/*f*.txt" | to text | each { |it| <shortcut name>.lnk $it }`
(Presuming you have some `*f*.txt` files somewhere in/below the directory where you are executing this)
Will fail to execute.
6. Give a full path to the shortcut instead, E.G:
`glob "**/*f*.txt" | to text | each { |it| C:\<shortcut name>.lnk $it }`
Will succeed.

### Expected behavior

I expected nu to run the executable pointed at by a Windows shortcut which is in Windows environment Path, since it works if you refer to it with the full path to the shortcut.

### Configuration

I had originally written this and took screenshots with:                                                                                                                                 
| key                | value                                    |
| ------------------ | ---------------------------------------- |
| version            | 0.92.2                                   |
| branch             |                                          |
| commit_hash        | 2a08a18b26865a86c793d183b9b042220ecb733a |
| build_os           | windows-x86_64                           |
| build_target       | x86_64-pc-windows-msvc                   |
| rust_version       | rustc 1.77.2 (25ef9e3d8 2024-04-09)      |
| rust_channel       | 1.77.2-x86_64-pc-windows-msvc            |
| cargo_version      | cargo 1.77.2 (e52e36006 2024-03-26)      |
| build_time         | 2024-04-10 21:24:37 +00:00               |
| build_rust_channel | release                                  |
| allocator          | mimalloc                                 |
| features           | default, sqlite, trash, which            |
| installed_plugins  |                                          |

But I upgraded to:
| key                | value                                    |
| ------------------ | ---------------------------------------- |
| version            | 0.95.0                                   |
| major              | 0                                        |
| minor              | 95                                       |
| patch              | 0                                        |
| branch             |                                          |
| commit_hash        | f93c6680bd5ae158190556eb1afc924b0290b52f |
| build_os           | windows-x86_64                           |
| build_target       | x86_64-pc-windows-msvc                   |
| rust_version       | rustc 1.77.2 (25ef9e3d8 2024-04-09)      |
| rust_channel       | 1.77.2-x86_64-pc-windows-msvc            |
| cargo_version      | cargo 1.77.2 (e52e36006 2024-03-26)      |
| build_time         | 2024-06-25 18:42:56 +00:00               |
| build_rust_channel | release                                  |
| allocator          | mimalloc                                 |
| features           | default, sqlite, system-clipboard, trash |
| installed_plugins  |                                          |

And verified problem persists.

**Comment by fdncred** [comment]
nushell doesn't know what windows shortcuts are, which is why it doesn't work. nushell uses the `open` crate in the `start` command. https://github.com/Byron/open-rs it doesn't look like it knows what windows shortcuts are either.

**Comment by JonathanBerkeley** [comment]
Hi @fdncred, thanks for the reply. Is it not unusual then that the shortcut was working, but only if the entire path is specified in version 0.92.2? I understand that doesn't work now in 0.95.0, but it appeared to be a bug to me due to one working and not the other.

**Comment by fdncred** [comment]
yes, it is unusual that it would work with full path. sounds like an `open` crate bug. I think if we want this to work it's best to have it added upstream.
